### ISSUE TITLE
Using Node.date causes segfault on Linux

### ISSUE THREAD

**Issue by C0deH4cker** [vapor/node#45]
With the current stable versions of Foundation on Linux, the [ISO 8601 calendar as used by `Node.date`](https://github.com/vapor/node/blob/4ad26cb05718cba1a860a1ccd921ad29772fdaa8/Sources/Node/Convertible/Date%2BConvertible.swift#L98) is not available. This causes `CFCalendarCreateWithIdentifier()` to return `NULL` and then crash in `CFCalendarSetLocale()` with a segfault. In GDB, it looks like this:

```
Thread 3 "App" received signal SIGSEGV, Segmentation fault.
0x00007ffff6c08b02 in CFCalendarSetLocale () from /home/user/.swiftenv/versions/3.0.2/usr/lib/swift/linux/libFoundation.so
```

The good news is that support for the ISO 8601 calendar in `CFCalendarCreateWithIdentifier()` has been added to master of https://github.com/apple/swift-corelibs-foundation. The bad news is the [commit that added support](https://github.com/apple/swift-corelibs-foundation/commit/57b4123b4e26ae926a3e9b2fdc6bf72f243e1e25) was published only 12 days ago, so it may take some time before this appears in a stable release of Swift. Hopefully a workaround for this bug on Linux can be added before this feature officially ships in Vapor 2.0.

**Comment by tanner0101** [comment]
Just ran into this last night and tracked it down to ISO 8601 as well. Thanks for the additional info about the status in corelibs!

**Comment by loganwright** [comment]
Fixed #56
